draft-ietf-quic-transport.md

```diff
@@ -670,7 +670,8 @@ field for a 0-RTT packet is selected by the client.
 The client can send 0-RTT packets after having received a packet from the server
 if that packet does not complete the handshake.  Even if the client receives a
 different connection ID from the server, it MUST NOT update the connection ID it
-uses for 0-RTT packets.  This enables consistent routing for all 0-RTT packets.
+uses for 0-RTT packets.  This ensures a predictable origin of connection IDs for
+all 0-RTT packets, see {{connection-id}}.
```

Comment:
This phrasing doesn't make sense to me.

The point is that all 0-RTT are labelled with the same connection ID, therefore they (should) end up in the same place.

Reply:
What I meant to say is that we are trying to make all packets of a particular type to always have CCID or SCID.  Mentioning "routing" here seems out of place -- network elements can use CID for whatever purpose they like, including routing.

I'll reword this.